cypress/e2e/signIn.cy.js

```diff
@@ -29,6 +28,10 @@ describe('Sign In page', () => {
   });
 
   it('should not provide an ability to log in with wrong credentials', () => {
+    signInPage.typeEmail('Gar' + user.email);
```

Comment:
When testing with incorrect credentials, ensure that the email format remains valid. Consider using a more realistic invalid email format to better simulate user error.